tools/promtail.sh

```diff
@@ -4,10 +4,11 @@ INSTANCEID="${1:-}"
 APIKEY="${2:-}"
 INSTANCEURL="${3:-}"
 NAMESPACE="${4:-default}"
-CONTAINERROOT="${5-/var/lib/docker}"
+CONTAINERROOT="${5:-/var/lib/docker}"
+PARSER="${6:-docker}"
 
-if [ -z "$INSTANCEID" -o -z "$APIKEY" -o -z "$INSTANCEURL" -o -z "$NAMESPACE" -o -z "$CONTAINERROOT" ]; then
-    echo "usage: $0 <instanceId> <apiKey> <url> <namespace> <container_root_path>"
+if [ -z "$INSTANCEID" -o -z "$APIKEY" -o -z "$INSTANCEURL" -o -z "$NAMESPACE" -o -z "$CONTAINERROOT" -o -z "$PARSER" ]; then
+    echo "usage: $0 <instanceId> <apiKey> <url> [<namespace:default>] [<container_root_path:/var/lib/docker>] [<parser:docker>]"
```

Comment:
My only concern here is that this notation implies that you can leave any of the options out and it'll figure out what you mean, which isn't the case.  Technically the correct notation would be something like `[<namespace>[<container_root_path> [<parser>]]]` but that's probably even more confusing to the average user.  On balance I think this is fine.

Reply:
I couldn't remember off my head a better way to indicate the optional values are "chained", so I updated it the way you have it, I think it makes more sense.